js/src/pages/ProjectPage.tsx

```diff
@@ -72,7 +70,7 @@ export const ProjectPage: React.FC = () => {
     fetchProject(slug);
   }, [location, slug, fetchProject]);
 
-  useClearState(clearStateAction);
+  if (!project || slug !== project.slug) return null;
```

Comment:
project after this line can't be null. So it's good idea to remove all `{!!project &&`

Reply:
done